infrastructure/src/main/java/org/corfudb/infrastructure/CompactorService.java

```diff
@@ -105,7 +107,6 @@ public CorfuRuntime getNewCorfuRuntime() {
         } catch (UnrecoverableCorfuError er) {
             log.error("Unable to connect to server due to UnrecoverableCorfuError: ", er);
             runtime.getParameters().getSystemDownHandler().run();
-            throw er;
```

Comment:
Note that this was dead code since the systemDownHandler here will always throw an `UnreachableClusterException`.